### PR TITLE
Flow improvements

### PR DESCRIPTION
**Pull request by 102**

### CODE REVIEW

File: es/index.js.flow [hunk 1]
```diff
@@ -12,10 +12,10 @@ declare type PropsT = {
   match: Match,
   history: RouterHistory,
   location: Location,
-  renderIfNotActive: bool,
+  renderIfNotActive?: bool,
   when: bool | (Location, ?Location) => bool,
-  disableNative: bool,
-  allowGoBack: bool,
+  disableNative?: bool,
+  allowGoBack?: bool,
```
Comment (102) [inline es/index.js.flow]: The missing props are both allowed in the code & in the typescript lib defs

File: test/src/components/issues/Issue11.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
-/* @flow */
```
Comment (102) [inline test/src/components/issues/Issue11.js]: test lib is ignored in `.flowconfig`, but causes false-positive errors when included in users config

Comment (102) [reply test/src/components/issues/Issue11.js]: it also probably makes sense to add `test` lib to `.npmignore` to avoid publishing it to npm at all

**Comment by ZacharyRSmith** [comment]
thx!  I think i added .npmignore correctly, and pub'd to npm

**Pull request merged**